galasa-ui/src/components/HomeContent.tsx

```diff
@@ -25,14 +30,28 @@ export default function HomeContent({markdownContentPromise}: HomeContentProps)
     const setRenderedHtmlFromMarkdown = async () => {
       try {
         const markdownContent = await markdownContentPromise;
-        setRenderedHtmlContent(md.render(markdownContent));
+
+        if (typeof markdownContent === "string") {
```

Comment:
wouldn't a switch statement be better here, with an error if markdown content was neither a string nor a 403 error ?